compute/cloud-client/src/main/java/compute/CreateInstanceWithExistingDisks.java

```diff
@@ -0,0 +1,119 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package compute;
+
+// [START compute_instances_create_with_existing_disks]
+
+import com.google.cloud.compute.v1.AttachedDisk;
+import com.google.cloud.compute.v1.Disk;
+import com.google.cloud.compute.v1.DisksClient;
+import com.google.cloud.compute.v1.InsertInstanceRequest;
+import com.google.cloud.compute.v1.Instance;
+import com.google.cloud.compute.v1.InstancesClient;
+import com.google.cloud.compute.v1.NetworkInterface;
+import com.google.cloud.compute.v1.Operation;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class CreateInstanceWithExistingDisks {
+
+  public static void main(String[] args)
+      throws IOException, ExecutionException, InterruptedException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+
+    // Project ID or project number of the Cloud project you want to use.
+    String projectId = "YOUR_PROJECT_ID";
+
+    // Name of the zone to create the instance in. For example: "us-west3-b"
+    String zone = "europe-central2-b";
+
+    // Name of the new virtual machine (VM) instance.
+    String instanceName = "YOUR_INSTANCE_NAME";
+
+    // Array of disk names to be attached to the new virtual machine.
+    // First disk in this list will be used as the boot disk.
+    List<String> diskNames = new ArrayList<>();
```

Comment:
Should we have a list of names here?  I think we should add some examples here for the user to change. If not it looks as if an empty list is okay.

```suggestion
    List<String> diskNames = List.of("your-boot-disk", "another-disk1", "another-disk2");
```

Reply:
That's a great suggestion. I've added it. Thanks.